pkg/apis/scheduling/types.go

```diff
@@ -261,6 +261,8 @@ type QueueSpec struct {
 	Capability v1.ResourceList
 	// State controller the status of queue
 	State QueueState
+	// UnReclaimAble indicate whether the queue can be reclaimed by other queue
+	UnReclaimAble bool
```

Comment:
let's use `Reclaimable` directly :)

Reply:
I assume you want to make it compatible with old versions. 
make it a pointer type will be ok